meteor/client/ui/RundownList/CreateTestingRundownPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { memo, useEffect, useState } from 'react'
+import { CreateTestingRundownOption } from '../../../lib/api/showStyles'
+import { MeteorCall } from '../../../lib/api/methods'
+import { UserAction, doUserAction } from '../../../lib/clientUserAction'
+import { useTranslation } from 'react-i18next'
+
+export const CreateTestingRundownPanel = memo(function CreateTestingRundownPanel(): JSX.Element | null {
+	const { t } = useTranslation()
+
+	const [options, setOptions] = useState<CreateTestingRundownOption[]>([])
+
+	useEffect(() => {
+		let isDisposed = false
+
+		const doPoll = () => {
+			MeteorCall.showstyles
+				.getCreateTestingRundownOptions()
+				.then((newOptions) => {
+					if (isDisposed) return
+					setOptions(newOptions)
+				})
+				.catch((e) => {
+					console.error(`Failed to get create testing rundown options: ${e}`)
+				})
+		}
+
+		const interval = setInterval(doPoll, 30000)
+		doPoll()
+
+		return () => {
+			isDisposed = true
+			clearInterval(interval)
+		}
```

Comment:
Not a fan of this polling.
Can't we expose this as a publication instead? Or possibly just calculate it client side, since it just uses the studio, showStyleBase and showStyleVariant?

Reply:
My thinking was that this is not worth the effort/complexity of a publication, as it does not need to be at all reactive it will only ever change when configuring Sofie, and it is cheap to calculate so doesn't benefit from any shared computation.
So it didn't feel worth the effort of setting up and building a full publication
I wouldn't be opposed to dropping the polling and having it loaded only on component mount

> Or possibly just calculate it client side

We currently only load the showstyles for the current rundowns, so the data isn't already there client side.

Reply:
Yeah, perhaps dropping the polling would be good enough